frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -20,14 +20,20 @@ import without from "lodash/without"
 import { withTheme } from "emotion-theming"
 import { WidgetStateManager, Source } from "src/lib/WidgetStateManager"
 import { MultiSelect as MultiSelectProto } from "src/autogen/proto"
-import { TYPE, Select as UISelect, OnChangeParams } from "baseui/select"
+import {
+  TYPE,
+  Select as UISelect,
+  Option,
+  OnChangeParams,
+} from "baseui/select"
 import {
   StyledWidgetLabel,
   StyledWidgetLabelHelp,
 } from "src/components/widgets/BaseWidget"
 import TooltipIcon from "src/components/shared/TooltipIcon"
 import { Placement } from "src/components/shared/Tooltip"
 import { VirtualDropdown } from "src/components/shared/Dropdown"
+import { fuzzyFilterSelectOptions } from "src/components/shared/Dropdown/Selectbox"
```

Comment:
Not sure if this should be pulled out into a utility or anything, only used in two modules.

Reply:
If we find ourselves using this any more then I'd say we should move it into a utility, but if it's only being used in two places I think it's fine to just import it from `Selectbox` for now